reflex/utils/prerequisites.py

```diff
@@ -24,25 +24,30 @@
 from reflex.config import get_config
 from reflex.utils import console, path_ops
 
+IS_WINDOWS = platform.system() == "Windows"
 
-def check_node_version(min_version=constants.MIN_NODE_VERSION):
-    """Check the version of Node.js.
 
-    Args:
-        min_version: The minimum version of Node.js required.
+def check_node_version():
+    """Check the version of Node.js.
 
     Returns:
-        Whether the version of Node.js is high enough.
+        Whether the version of Node.js is valid.
     """
     try:
         # Run the node -v command and capture the output
         result = subprocess.run(
-            ["node", "-v"], stdout=subprocess.PIPE, stderr=subprocess.PIPE
+            ["node" if IS_WINDOWS else constants.NODE_PATH, "-v"],
```

Comment:
Doesn't the `constants` already to this check?

Reply:
ah yeah, missed that, will update this